server/src/main/java/org/tbbtalent/server/request/candidate/occupation/UpdateCandidateOccupationRequest.java

```diff
@@ -16,33 +16,15 @@
 
 package org.tbbtalent.server.request.candidate.occupation;
 
+import lombok.Getter;
```

Comment:
This cosmetic change should really have been in a separate commit - allows reviewer to just focus on functional changes